catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -82,12 +89,14 @@
 
     public EndpointUtil(List<MetacardType> metacardTypes, CatalogFramework catalogFramework,
             FilterBuilder filterBuilder, List<InjectableAttribute> injectableAttributes,
-            AttributeRegistry attributeRegistry) {
+            AttributeRegistry attributeRegistry,
+            ConfigurationApplication configurationApplication) {
         this.metacardTypes = metacardTypes;
         this.catalogFramework = catalogFramework;
         this.filterBuilder = filterBuilder;
         this.injectableAttributes = injectableAttributes;
         this.attributeRegistry = attributeRegistry;
+        this.querySize = configurationApplication.getResultCount();
```

Comment:
✏️  / ❗️  Just a heads up, since this is instantiated in the constructor it will use the value stored in the configuration but will **not** update when it is changed. We can just create a ticket to fix it in another PR if not done in this PR.